### DISCUSSION TITLE
Document built together

### DISCUSSION THREAD

**Discussion by damonyoungcc** [d3/d3#3736] [Q&A]
I have noticed that d3 has updated its new website and documentation, which is amazing. Regarding the documentation, I would like to ask if it can be open source so that the community can be built together. As a Chinese developer, I voluntarily participate in the translation of Chinese documents to provide convenience for more Chinese developers who love d3.

**Comment by Fil** [comment]
The documentation is open-source, see the https://github.com/d3/d3/tree/main/docs directory. To test it locally, simply run `yarn docs:dev`.

**Reply by damonyoungcc** [reply]
That's awesome!
